controllers/operands/aaq_test.go

```diff
@@ -204,7 +204,7 @@ var _ = Describe("AAQ tests", func() {
 		})
 
 		It("should create AAQ if the applicationAwareConfig field exists", func() {
-			hco.Spec.ApplicationAwareConfig = &v1beta1.ApplicationAwareConfigurations{}
+			hco.Spec.FeatureGates.EnableApplicationAwareQuota = ptr.To(true)
```

Comment:
the test case description doesn't match what we actually checking (`applicationAwareConfig` vs. the FG)

Reply:
Will fixed in future PR